### PR TITLE
[PTDT-2553] Added integration tests for MMC MAL/GT imports

### PR DESCRIPTION
**Pull request by real-mjozwiak**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document change (fix typo or modifying any markdown files, code comments or anything in the examples folder only)

## All Submissions

- [ ] Have you followed the guidelines in our Contributing document?
- [ ] Have you provided a description?
- [ ] Are your changes properly formatted?

## New Feature Submissions

- [ ] Does your submission pass tests?
- [ ] Have you added thorough tests for your new feature?
- [ ] Have you commented your code, particularly in hard-to-understand areas?
- [ ] Have you added a Docstring?

## Changes to Core Features

- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully run tests with your changes locally?
- [ ] Have you updated any code comments, as applicable?

### CODE REVIEW

File: libs/labelbox/tests/data/annotation_import/conftest.py [hunk 1]
```diff
@@ -696,6 +768,42 @@ def _create_prompt_response_project(client: Client, rand_gen, media_type, normal
     
     return prompt_response_project, ontology
 
+def _create_offline_mmc_project(client: Client, rand_gen, data_row_json, normalized_ontology) -> Tuple[Project, Ontology, Dataset]:
+    """ Shared function to configure project for integration tests """
```
Comment (Gabefire) [inline libs/labelbox/tests/data/annotation_import/conftest.py]: Can you update this doc string to something more accurate or remove it altogether?

**Review by Gabefire — COMMENTED** [review]
Let's great!

File: libs/labelbox/tests/data/annotation_import/conftest.py (hunk 1 above)
Comment (real-mjozwiak) [reply libs/labelbox/tests/data/annotation_import/conftest.py]: oh yes, of course

**Review by Gabefire — DISMISSED** [review]

File: libs/labelbox/tests/data/annotation_import/conftest.py [hunk 2]
```diff
@@ -194,6 +193,21 @@ def llm_human_preference_data_row(global_key):
     return llm_human_preference_data_row
 
 
+@pytest.fixture(scope="module")
```
Comment (vbrodsky) [inline libs/labelbox/tests/data/annotation_import/conftest.py:156]:
```suggestion
@pytest.fixture(scope="session")
```

File: libs/labelbox/tests/data/annotation_import/conftest.py [hunk 3]
```diff
@@ -194,6 +193,21 @@ def llm_human_preference_data_row(global_key):
...
+def mmc_data_row_url():
+    return "https://storage.googleapis.com/labelbox-datasets/conversational_model_evaluation_sample/offline-model-chat-evaluation.json"
+
+
+@pytest.fixture(scope="module", autouse=True)
```
Comment (vbrodsky) [inline libs/labelbox/tests/data/annotation_import/conftest.py:161]: why `autouse=True` ? doesn't it mean the fixture will be request by each and every test in annotation_import?

File: libs/labelbox/tests/data/annotation_import/conftest.py [hunk 4]
```diff
@@ -730,6 +802,41 @@ def _create_prompt_response_project(
     return prompt_response_project, ontology
 
 
+def _create_offline_mmc_project(client: Client, rand_gen, data_row_json, normalized_ontology) -> Tuple[Project, Ontology, Dataset]:
```
Comment (vbrodsky) [inline libs/labelbox/tests/data/annotation_import/conftest.py]: Why is this not a fixture? How do you intend to clean resources after the test if it is not a fixture?

File: libs/labelbox/tests/data/annotation_import/conftest.py (hunk 3 above)
Comment (real-mjozwiak) [reply libs/labelbox/tests/data/annotation_import/conftest.py:161]: I was inspired by what @Gabefire did on other similar fixtures. Definitely doesn't have to be autouse, but I guess it makes the fixtures run once at session start and cache results?

File: libs/labelbox/tests/data/annotation_import/conftest.py (hunk 4 above)
Comment (real-mjozwiak) [reply libs/labelbox/tests/data/annotation_import/conftest.py]: This is done in the same fashion as the other existing `_create_*_project` functions. They are used by `configured_project` fixture which does the cleaning.

File: libs/labelbox/tests/data/annotation_import/conftest.py (hunk 2 above)
Comment (real-mjozwiak) [reply libs/labelbox/tests/data/annotation_import/conftest.py:156]: Could do that, but then we could also do it for all the other similar fixtures. What do you think?

**Review by vbrodsky — DISMISSED** [review]

**Review by vbrodsky — APPROVED** [review]

**Pull request merged**